client/state/preferences/actions.js

```diff
@@ -28,26 +28,35 @@ export function fetchPreferences() {
 }
```

Comment:
- They are extremely rarely updated, which leads to comments that lie
- Information they provide is often useless because it just repeats the parameters and we are not using them to generate any documentation
- They encourage bad code because they obscure the fact that code underneath is bad